utils/tests_fetcher.py

```diff
@@ -1064,10 +1064,17 @@ def infer_tests_to_run(
     impacted_files = sorted(set(impacted_files))
     print(f"\n### IMPACTED FILES ###\n{_print_list(impacted_files)}")
 
+    model_impacted = sorted({"/".join(x.split("/")[:3]) for x in impacted_files if x.startswith("tests/models/")})
```

Comment:
We shouldn't need to sort again here, I don't think

Reply:
no need indeed - because we only need it to be a set instead of a list.